crypto/src/main/kotlin/web5/sdk/crypto/Dsa.kt

```diff
@@ -12,6 +12,21 @@ public enum class JwaCurve {
   Ed25519;
 
   public companion object {
+
+    /**
+     * Convert JwaCurve nimbusds JWK curve.
+     * Used to temporarily bridge the gap between moving from nimbusds JWK methods
+     * to rolling our own JWK methods
+     * @param curve
+     * @return nimbus JWK Curve
```

Comment:
clarification: I thought we're trying not to return nimbus objects. Is this an exceptional case or i have I misunderstood the goal?

Reply:
this is an exceptional case, thanks for bringing it up!

currently, we still have to use nimbusds internally to create the private key material. there's a follow-up issue to completely eject from nimbusds and write the private key ourselves by using google tink library: https://github.com/TBD54566975/web5-kt/issues/273

this method is still used in `AwsKeyManager#getPublicKey`, where we take `java.security.PublicKey` and convert it to an ECPublicKey, which is one way we create privatekey using nimbusds library. 

we "ejected from nimbusds" in this PR in the sense that we no longer expose nimbusds' JWK type as part of the library API surface.

i expect that we can safely get rid of this method once we work on that follow-up issue.